internal/kms/README.md

```diff
@@ -74,109 +75,30 @@ Supported are:
 
-To use the AWS S3 Bucket plugin, you need to have an existing [AWS account](https://aws.amazon.com/de/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+For authentication an access key ID and an access key secret is used.
+As a fallback, the client may try to automatically fetch the data from the local AWS directory.
 
-For authentication, you have to pass a config file to the plugin. The AWS config package lets you automatically fetch the data from the local AWS directory.
+#### Azure Blob Storage
 
-#### Passing credentials automatically
-
-You need to store your credentials in your local AWS directory at `$HOME/.aws/`. The AWS config package uses the values from the directory to build a config file, which is used to authenticate the client. The local AWS directory must contain two files:
-
-* `credentials`
-
-    ```bash
-    [default]
-    aws_access_key_id = MyAccessKeyId
-    aws_secret_access_key = MySecretAccessKey
-    ```
-
-* `config`
-
-    ```bash
-    [default]
-    region = MyRegion
-    output = json
-    ```
-
-If you have the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) installed, you can
-initialise the files with the following command:
-
-```bash
-aws configure
-```
-
-To create the client:
-
-```Go
-cfg, err := config.LoadDefaultConfig(context.TODO())
-store, err := storage.NewAWSS3Storage(context.TODO(), "bucketName", cfg, func(*s3.Options) {})
-```
-
-### Azure Blob Storage
-
-To use the Azure Blob storage plugin, you need to first [create a storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) or give your application access to an existing storage account.
-
-The plugin uses a connection string created for the storage account to authenticate itself to the Azure API.
-The connection string can be found in your storage account in the Azure Portal under the "Access Keys" section or with the following Azure CLI command:
-
-```bash
-az storage account show-connection-string -g MyResourceGroup -n MyStorageAccount
-```
-
-The client will use the specified Blob container if it already exists, or create it first otherwise.
-
-To create the client:
-
-```Go
-connectionString := "DefaultEndpointsProtocol=https;AccountName=<myAccountName>;AccountKey=<myAccountKey>;EndpointSuffix=core.windows.net"
-store, err := storage.NewAzureStorage(context.TODO(), connectionString, "myContainer", nil)
-```
+Authorization for Azure Key Vault happens through the use of manged identities.
+The managed identity requires the following permissions:
 
-### Google Cloud Storage
+* `Microsoft.Storage/storageAccounts/blobServices/containers/write` (only if a storage container is not set up in advance)
+* `Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write`
+* `Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read`
+* `Microsoft.Storage/storageAccounts/blobServices/containers/blobs/add/action`
```

Comment:
You also need these on the storage account you are using:
- Storage Blob Data Contributor
- Storage Queue Data Contributor

Reply:
I'm not 100% sure about how exactly Azure IAM works, but I think `Storage Blob Data Contributor` and `Storage Queue Data Contributor` are roles that provide permissions to access storage accounts, e.g. `Data Contributor` provides the `Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write` permission